nucliadb/src/nucliadb/common/cluster/manager.py

```
@@ -104,6 +104,14 @@ async def apply_for_all_shards(
             errors.capture_exception(exc)
             raise NodeError("Node unavailable for operation") from exc
 
+        for result in results:
+            if isinstance(result, Exception):
+                errors.capture_exception(result)
+                raise NodeError(
```

I think this is a good idea. Too easy to ignore error otherwise.